templates/agent-conf.d/http_check.yaml.erb

```diff
@@ -28,7 +28,9 @@ instances:
 <% if instance['collect_response_time'] -%>
     collect_response_time: <%= instance['collect_response_time'] %>
 <% end -%>
-    disable_ssl_validation: <%= instance['disable_ssl_validation'] %>
+<% if instance['disable_ssl_validation'] -%>
```

Comment:
I tested successfully with this config in the manifest:
```
node 'ip-0-0-0-0.ec2.internal' {
    class { "datadog_agent":
        api_key => "key"
    }

    class { 'datadog_agent::integrations::http_check':
      instances => [{
        'sitename'  => 'google',
        'url'       => 'http://www.google.com',
        'tags'      => 'site:google'
      },
      {
        'sitename'  => 'datadog',
        'url'       => 'http://www.datadoghq.com',
        'tags'      => 'site:dog'
      }]
    }
}
```
We only add parameters when they are found. Another alternative would be to use `if instance.has_key?("disable_ssl_validation")` although I used the above for consistency. What do you think?